templates/acs-deployment-master.yaml

```diff
@@ -243,6 +258,37 @@ Parameters:
       AllowedValues:
       - "true"
       - "false"
+    MQInstanceType:
+      AllowedValues:
+        - "mq.t2.micro"
+        - "mq.m4.large"
+        - "mq.m5.large"
+        - "mq.m5.xlarge"
+        - "mq.m5.2xlarge"
+        - "mq.m5.4xlarge"
+      ConstraintDescription: Must contain valid AmazonMQ Host instance type
+      Default: mq.m5.large
+      Description: The broker's instance type
+      Type: String
+    MQDeploymentMode:
+      AllowedValues:
+        - "SINGLE_INSTANCE"
+        - "ACTIVE_STANDBY_MULTI_AZ"
+      ConstraintDescription: Must contain valid AmazonMQ deployment mode
+      Default: ACTIVE_STANDBY_MULTI_AZ
```

Comment:
Does this *have* to be the default, could we not default to SINGLE_INSTANCE (which I presume would be cheaper) so that devs/tests are minimising costs?

Reply:
Yes using multi AZ doubles the costs for AmazonMQ. Jan suggested to focus on production environments.

Reply:
OK, I guess the "proper" way to do this anyway is to have a "Stage/Environment" parameter for dev/test/prod and that can determine the "size" defaults for several resources.

Reply:
I suggested we have sensible defaults. However, it should be possible to override when deploying the stack, eg. drop-downs for:

* broker deployment mode
* broker instance type

Reply:
I'm ok with current PR, ie. some realistic defaults with option to override (eg. to scale up or down, as needed).